rasa/server.py

```diff
@@ -670,7 +670,7 @@ def create_app(
                 domain=domain_path,
                 config=config_path,
                 training_files=temp_dir,
-                output_path=rjs.get("out", DEFAULT_MODELS_PATH),
+                output_path=rjs.get("out", tempfile.gettempdir()),
```

Comment:
this is fine but why not use the existing `temp_dir` earlier?

Reply:
good point, updated it 👍 have you seen https://github.com/RasaHQ/rasa/pull/4480#issuecomment-532604420 ?

Reply:
oh sorry no I didn't 😄 see the other PR I just requested review from you for the rasa-x part

Reply:
but yes good point let's take it out

Reply:
pushed the change